webcompat/views.py

```diff
@@ -302,13 +302,18 @@ def cssfixme():
 @app.errorhandler(500)
 def custom_error_handler(err):
     # log the exception stack trace
-    app.logger.exception("!!!")
-    if api_call(request):
-        return api_message(err.code)
-    return render_template(
-        'error.html',
-        error_code=err.code,
-        error_message=ERROR_DICT[err.code]), err.code
+    # (but don't bother for localhost because the
+    # Flask debugger is already enabled)
+    if not app.config['LOCALHOST']:
+        app.logger.exception("Exception thrown:")
+    try:
+        if api_call(request):
+            return api_message(err.code)
+        return render_template('error.html', error_code=err.code,
+                               error_message=ERROR_DICT[err.code]), err.code
+    except AttributeError:
+        # Somethign bad happened, we're not dealing with an HTTPError
+        abort(500)
```

Comment:
Do we want to log somewhere the Exception?
And this seems recursive. abort(500) will call the custom error handler. Basically called twice?

Note that I don't have a better idea for now.